web-access-app/src/main/webapp/WEB-INF/jsp/fullRecord/adminUnit.jsp

```diff
@@ -42,43 +42,48 @@
 
 <c:url var="fullRecordUrl" scope="page" value="record/${briefObject.id}" />
 <div class="contentarea">
-    <div id="is-admin-unit" class="columns browse-header">
```

Comment:
I am guessing all the changes in this file are converting the admin unit page over to following the same layout as the other pages, rather than changes directly related to the access request buttons? So much reformatting, but seems necessary.

One thing we should update though, it looks like before these changes, if I shrink my browser down to mobile widths, all the parts properly display in rows. But after these changes the breadcrumbs are getting smushed into the left hand side, and all the rest of the content is on the right. This is happening for AdminUnit and Collection pages, but not folder, work or file.